Scenarios/TESTScenarios.json

```diff
@@ -27,7 +27,7 @@
   ],
   "scenarios": [
     {
-      "name": "GQ -> C(45%) -> GQ -> C(50%)",
+      "name": "PageView",
```

Comment:
No, the scenarios are in the repository as examples, good practice would be not to depend on them.

(I'm guilty of this too, but I am trying to move away from it, I wanted to put mine there just so people would have examples of what a scenario file would look like.)

Maybe I shall rename the folder examples.